ShippingApp/src/main/java/com/javaschool/queue/QueueSender.java

```diff
@@ -0,0 +1,108 @@
+package com.javaschool.queue;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.javaschool.common.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.AmqpTemplate;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class QueueSender {
+
+    private static final Logger logger = LoggerFactory.getLogger(QueueSender.class);
+
+    private AmqpTemplate rabbitTemplate;
+
+    private GlobalProperties globalProperties;
+
+    private ObjectMapper objectMapper;
+
+    public QueueSender(final AmqpTemplate rabbitTemplate, GlobalProperties globalProperties) {
+        this.rabbitTemplate = rabbitTemplate;
+        this.globalProperties = globalProperties;
+        this.objectMapper = new ObjectMapper();
```

Comment:
If you are creating an instance inside the constructor maybe there is no need to have this line here at all. But it should be better to follow the Dependency Injection pattern.